feat(words): add Python words related to Bokeh, Curses, Pandas, and ros-visualization

nabetetsu opened the pull request:
Add words related Bokeh, Curses, Pandas libraries and rqt_graph.

- Bokeh :https://docs.bokeh.org/en/latest/
    - "Bokeh"
    - "bokeh"
- Curses API: https://docs.python.org/3/library/curses.html#functions
    - "addstr"
    - "getch"
    - "getmaxyx"
      - https://docs.python.org/3/library/curses.html#curses.window.getmaxyx
    - "scrollok"
      - https://docs.python.org/3/library/curses.html#curses.window.scrollok
    - "NPAGE"
    - "PPAGE"
      - https://docs.python.org/3/library/curses.html#constants
    - "stdscr"
- Pandas: https://pandas.pydata.org/docs/reference/frame.html#dataframe
    - "dropna"
      - https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.dropna.html#pandas-dataframe-dropna
    - "iterrows"
      - https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.iterrows.html#pandas-dataframe-iterrows
- rqt_graph: 
    - "rosviz"
        - https://github.com/ros-visualization/rqt_graph/blob/92b9278a1d2a2d7eabc115bc5927afac803b066a/src/rqt_graph/rosgraph2_impl.py#L59
    - "rtype"
        - https://github.com/ros-visualization/rqt_graph/blob/92b9278a1d2a2d7eabc115bc5927afac803b066a/src/rqt_graph/rosgraph2_impl.py#L161:
- Python collections:
  - "popleft"
      - https://docs.python.org/ja/3/library/collections.html#collections.deque.popleft

kenji-miyake commented:
@nabetetsu Could you confirm the following words? I couldn't find them.
- "getmaxyx"
- "scrollok"
- "NPAGE"
- "popleft"

Also, please update the link of pandas so that reviewers can find the words from the link.

nabetetsu commented:
@kenji-miyake Thanks for the review.

> * "getmaxyx"
> * "scrollok"
> * "NPAGE"
I have updated the original comment with the link to the API document page.
Also, I forgot to add "NPAGE" and "PPAGE" to the dictionary. I will update the branch right after this.

> * "popleft"
It's my mistake. This is the API in the python collections module. I have updated the link in the comment.

> Also, please update the link of pandas so that reviewers can find the words from the link.
